### ISSUE TITLE
Develop a schema for `static-sector` interface plugins

### ISSUE THREAD

**Issue by jsolbrig** [NRLMMD-GEOIPS/geoips#110]
# Requested Update

### Description
Agree upon and finalize a schema for `static-sectors` interface plugins. This schema should include the base interface schema.

This schema is a little more complicated than `boundaries` and `gridlines`. It may require a little more thought to determine which parameters should actually be exposed.

#### Questions
- Should we allow resetting `a`? Is there usefulness there?
- Should we allow setting `units`? I've never seen this used.
- What is `resolution` here?
- What rules do we define for the `sector_info` components? E.g. no dashes? No underscores?
- Do we really need to define `area_extent` to define the domain for a sector? There used to be a way to do this via LL and UR lat/lon coordinates.

Proposed Schema (needs work!):
```yaml
$ref: base-schema
spec:
  type: object
  additionalProperties: false
  properties:
    projection:
      type: object
      properties:
        a:
          type: number
          default: 6371228.0
        lat_0:
          type: number
        lon_0:
          type: number
        proj:
          type: string
          enum: [a list of available projections]
        units:
          type: string
          default: m
    resolution:
      type: object
      properties:
        x:
          type: integer
        y:
          type: integer
    sector_info:
      type: object
      properties:
        continent:
          type: string
        country:
          type: string
        area:
          type: string
        subarea:
          type: string
        state:
          type: string
        city:
          type: string
    shape:
      type: object
      properties:
        height:
          type: integer
        width:
          type: integer
    area_extent:
      type: object
      properties:
        lower_left_xy:
          type: array
          minItems: 2
          maxItems: 2
        upper_right_xy:
          type: array
          minItems: 2
          maxItems: 2
```

### Background and Motivation
*  [ OPTIONAL: Provide additional detailed information and background related to this issue ]

### Alternative Solutions
*  [ OPTIONAL: If appropriate, list alternative fixes that could be considered]
  
### Environment
*  [ IF APPROPRIATE: Corresponding repository versions ]

### Code to demonstrate issue
*  [ IF APPROPRIATE: Can include steps here to identify/demonstrate bug fix or desired feature ]

### Checklist for Completion
* [ ]

**Comment by jsolbrig** [comment]
Which is deprecated? `resolution` or `shape`? @mindyls

**Comment by jsolbrig** [comment]
This is done and will be merged in #260
